tools/src/test/scala/scala/scalanative/linker/package.scala

```diff
@@ -0,0 +1,13 @@
+package scala.scalanative
+
+import scala.scalanative.nir._
+
+package object linker {
+  val linktimeInfoProperties = "scalanative.meta.linktimeinfo"
```

Comment:
```suggestion
  val linktimeInfoProperties = "scala.scalanative.meta.linktimeinfo"
```
should fix the build errors.